crux-llvm/src/Crux/LLVM/Config.hs

```diff
@@ -161,6 +162,9 @@ llvmCruxConfig = do
          loopMerge <- Crux.section "opt-loop-merge" Crux.yesOrNoSpec False
                         "Insert merge blocks in loops with early exits (i.e. breaks or returns). This may improve simulation performance."
 
+         debugIntrinsics <- Crux.section "debug-intrinsics" Crux.yesOrNoSpec False
```

Comment:
I'm unclear on what determines if something deserves to be both a Crux configuration file option and a command-line option. For now, I went with making `debug-intrinsics` solely a configuration file option, but I can also make it a command-line option if you prefer.

Reply:
I don't think we have a principled rule, just a gut feeling about which things are common enough to be worth putting in command line options.  I think this is find as a config-file only option.